components/server/src/messaging/redis-subscriber.ts

```diff
@@ -172,8 +210,13 @@ export class RedisSubscriber implements LocalMessageBroker {
     }
 
     listenForPrebuildUpdatableEvents(listener: HeadlessWorkspaceEventListener): Disposable {
-        // TODO: not implemented
-        return Disposable.create(() => {});
+        // we're being cheap here in re-using a map where it just needs to be a plain array.
```

Comment:
i did not understand that comment as I don't see a map being used.

Reply:
This is a direct copy-paste from the existing local-messagebus.ts. Basically, we don't track the listeners by an ID for the Prebuild Udpatable, but we re-use a map, with a single key to re-use the helper for creating the listeners. I'll clean this up after we get rid of messagebus, but wanted to keep it as close to the current implementation to reduce chance for mistakes